conf/messages

```diff
@@ -1275,126 +1275,107 @@ reliefDetails.cyaChange.lettingsReliefValue=Amount of Lettings Relief
 reliefDetails.cyaChange.otherReliefsName=Other reliefs names
 reliefDetails.cyaChange.otherReliefsAmount=Other reliefs amount
 
-
-
 exemptionAndLosses.caption=Losses and exemptions
 
 inYearLosses.title=Do you want to include any other Capital Gains Tax losses made in tax year {0} to {1}?
+inYearLosses.agent.title=Do you want to include any other Capital Gains Tax losses made in tax year {0} to {1}?
```

Comment:
is this right? Would we not say something like "Does you client want to include..."?